ARImageTracking/ContentView.swift

```diff
@@ -47,9 +47,13 @@ struct ARViewContainer: UIViewRepresentable {
             let playerItem = AVPlayerItem(url: videoURL)
             videoPlayer = AVPlayer(playerItem: playerItem)
             let videoMaterial = VideoMaterial(avPlayer: videoPlayer)
-            
+
+			// size of video plane depending of the image
+			let width: Float = Float(imageAnchor.referenceImage.physicalSize.width * 1.1)
```

Comment:
could you change the value of 1.1 to 1.03 for both width and height? that seems to be the magic number for me after testing (for the video to fit the iPhone box perfectly)